packages/EasyDoctrine/src/Dispatchers/DeferredEntityEventDispatcher.php

```diff
@@ -147,7 +168,7 @@ public function enable(): void
      * @param string $oid
      * @param array<string, array{mixed, mixed}> $entityChangeSet
      *
-     * @return \EonX\EasyDoctrine\Events\EntityCreatedEvent|\EonX\EasyDoctrine\Events\EntityUpdatedEvent
+     * @return \EonX\EasyDoctrine\Events\EntityCreatedEvent|\EonX\EasyDoctrine\Events\EntityUpdatedEvent|\EonX\EasyDoctrine\Events\EntityDeletedEvent
```

Comment:
This change makes me think we should introduce something like `EntityActionEventInterface` and change this signature.

Reply:
Done.